src/deploy.js

```diff
@@ -27,14 +30,15 @@ const trackProgress = function(progressData) {
   };
 };
 
-module.exports = function(progressData, context, client, storage, config, slackTemplate) {
+module.exports = function(progressData, context, client, storage, config, slackTemplate, managementClientId) {
```

Comment:
Do we need to specify this? We should be able to retrieve it using `config('AUTH0_CLIENT_ID')` no?